front/app/containers/Admin/tools/Esri/index.tsx

```diff
@@ -31,17 +31,9 @@ const Esri = () => {
         h="240px"
         background={colors.teal200}
         display="flex"
-        justifyContent="center"
+        borderRadius="3px"
```

Comment:
@amanda-anderson Too late now, but better to use borderRadius defined on theme (also 3px).